versions.properties

```diff
@@ -99,6 +99,7 @@ version.com.jakewharton.retrofit..retrofit2-kotlin-coroutines-adapter=0.9.2
 
 version.androidx.viewpager2=1.0.0
 
+## unused
```

Comment:
Question: Why don't we remove this library if we are not using it?

Reply:
Ah good question; I'd been assuming it was a `refreshVersions` bug but upon closer look, I think define `turbine` twice (slightly differently). One probably is genuinely unused.

I'll tidy that up before merging. Good spot!